### PR TITLE
use the configured resource limits

### PR DESCRIPTION
**Pull request by teg**
It is the distro/admin's responsibility to ensure that these are reasonable.

### CODE REVIEW

File: src/launch/config.c [hunk 1]
```diff
@@ -1109,12 +1136,26 @@ static void config_parser_end_fn(void *userdata, const XML_Char *name) {
                 break;
         }
 
+        case CONFIG_NODE_LIMIT: {
+                unsigned long long limit;
+                char *end;
+
+                errno = 0;
+                limit = strtoull(state->current->cdata, &end, 10);
+                if (end != state->current->cdata && *end == '\0' && errno == 0)
```
Comment (dvdhrm) [inline src/launch/config.c]: Errrr, this hard-codes `sizeof(long long)` in user-visible stuff. This is 64bit everywhere, I know, so probably fine. But... eh... if "long long" is not 64bit, this suddenly has different maximums depending on the machine. Not sure that's what we want.

Can we make this a util/ helper? util_strtou64()? and then "static_assert(...)"?

Comment (teg) [reply src/launch/config.c]: Will do.

**Comment by teg** [comment]
Force-pushed. Fixes the saturating multiplication as well as uses 64-bit integers everywhere, rather than unsigned long long.

**Comment by dvdhrm** [comment]
Looks good to me! Thanks!

**Pull request merged**